liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -89,14 +98,70 @@ private static void initializeAppsLocation(File serverXML, File configDir) {
             parseApplication(doc, "/server/application");
             parseApplication(doc, "/server/webApplication");
             parseApplication(doc, "/server/enterpriseApplication");
-            parseInclude(doc, "/server/include");
+            parseInclude(doc);
             parseConfigDropinsDir();
 
         } catch (Exception e) {
             e.printStackTrace();
         }
     }
     
+    private static void initializeProperties(File serverXML, File configDir) {
+        
+        // Server variable precedence in ascending order if defined in multiple locations.
+        //
+        // 1. variables from '${server.config.dir}/server.env'
+        // 2. variables from '${server.config.dir}/bootstrap.properties' 
+        // 3. variables defined in <include/> files
+        // 4. variables from configDropins/defaults/<file_name>
+        // 5. variables defined in server.xml
+        //    e.g. <variable name="myVarName" value="myVarValue" />
+        // 6. variables from configDropins/overrides/<file_name>
+        
+        InputStream fis = null;
+        
+        try {
+            serverFile = serverXML;
+            configDirectory = configDir;
+            
+            Document doc = parseDocument(new FileInputStream(serverFile));
+
+            props = new Properties();
+            
+            Properties propServerEnv = new Properties();
+            
+            // get variables from server.env 
+            File serverEnvFile = new File(serverFile.getParentFile(), "server.env");
+            if (serverEnvFile.exists()) {
+                propServerEnv.load(new FileInputStream(serverEnvFile));
+                props.putAll(propServerEnv);
+            }
+            
+            Properties propBootStrap = new Properties();
+            
+            File bootstrapFile = new File(serverFile.getParentFile(), "bootstrap.properties");
```

Comment:
Fixed. Also, bootstrapProperties and bootstrapPropertiesFile are handled together by 'StartDebugMojoSupport.copyConfigFiles()'